nixos/modules/services/logging/logrotate.nix

```diff
@@ -112,7 +111,10 @@ in
 
   options = {
     services.logrotate = {
-      enable = mkEnableOption "the logrotate systemd service";
+      enable = mkEnableOption "the logrotate systemd service" // {
```

Comment:
This is debatable imo but since you are the one doing the contribution I prefer your opinion over mine :)

Reply:
Well, there's another argument if you want: we'll need to keep 'enable' as a compatibility setting anyway, so it doesn't hurt to keep? :P 

I don't particularly think my opinion is worth more than another's here, in case of disagreements like this if one side can't be convinced quickly I'll defer to a third opinion if any is given.

Reply:
Keep `enable`. At some point someone is going to come back to this PR and complain that `logrotate` is on their system. I promise. It will be good to tell them how easy it is to disable via `enable = false;`.